src/seed_tools/utils/study_validation.ts

```diff
@@ -80,11 +88,97 @@ function checkExperiments(study: Study): string[] {
     }
     experimentNames.add(experiment.name);
     totalProbability += experiment.probability_weight ?? 0;
+
+    // Validate features.
+    if (experiment.feature_association !== undefined) {
+      if (
+        experiment.feature_association.enable_feature !== undefined &&
+        experiment.feature_association.enable_feature.length > 0
+      ) {
+        for (const feature of experiment.feature_association.enable_feature) {
```

Comment:
WDYT about making `let features_to_check: string[] = []` and extending it.
Then just a one for-loop to check them all.